src/handlers/messageHandler.ts

```diff
@@ -28,10 +28,14 @@ export async function handleMessageCreate(message: Message): Promise<void> {
 	if (message.client.user === null) return;
 
 	if (message.system) return;
-	if (message.author.bot) return; // TODO: Make configurable, some users want this to be allowed (in certain channels) Also we probably need to check against ourselves if removed
 	if (!message.channel.isText()) return;
+	if (!message.inGuild()) return;
+	if (message.author == message.client.user) return;
 
-	if (message.channel.isThread()) {
+	const includeBots = includeBotsForAutothread(message.guild.id, message.channel.id);
+	if (!includeBots && message.author.bot) return;
```

Comment:
Check if the message was sent by the client (who knows if this can happen) but who knows at this point

Reply:
Check the line above good sir 😄

Reply:
In fact, it was even in a comment before I did it (that's how I remembered, lol)
https://github.com/MarcusOtter/discord-needle/pull/34/commits/379a73de966993cdb3c7361773f98b3c3d2222f5#diff-ed14967f4cd98459fed13422514b45f8e628d10e312b5047c4f8d13ffb0684c2L31